interface/logic-editor.md

```diff
@@ -1,40 +1,61 @@
-{% endtab %}
+* ![](/.gitbook/assets/interface/logic-editor/import_blueprint.PNG) **Import blueprint** icon allows you to import a **Logic Graph** saved on your computer into the **Logic Editor**.
+  
+* ![](/.gitbook/assets/interface/logic-editor/group.PNG) **Group** icon clusters **Logic Graph**s into distinct groupings. See a usage example below: ![](/.gitbook/assets/interface/logic-editor/group.gif) 
 
-{% tab title="3. Toolbox" %}
-![](/.gitbook/assets/logic-editor-toolbox.png)
+* ![](/.gitbook/assets/interface/logic-editor/highlightAndUnhighlight.PNG) **Highlight** icon applies a distinct color to sections of a **Logic Graph** that are connected together. **Remove highlight** reverses the action performed by clicking the **Highlight** icon. See a usage example below: ![](/.gitbook/assets/interface/logic-editor/highlightAndUnhighlight.gif) 
 
-Like the name suggests, the **Toolbox** tab contains all of the individual tools you will need to create your logic. The function and purpose of all the nodes is covered, in depth, in the **Toolbox** section and it is _highly_ recommended that you refer to it, to understand how each individual node functions. In short though, the **Toolbox** contains a categorized list of all nodes available at your disposal, which can be added to your logic by dragging and dropping an item into your **Logic Graph**.
-{% endtab %}
+* ![](/.gitbook/assets/interface/logic-editor/align.PNG) 
+**Align horizontally** and **Align Vertically** icons will align a **Node Graph** horizontally and vertically respectively in the **Logic Editor**. See usage example below: ![](/.gitbook/assets/interface/logic-editor/align-horizontally.gif) ![](/.gitbook/assets/interface/logic-editor/align-vertically.gif) 
 
-{% tab title="4. Variables" %}
-![](/.gitbook/assets/logic-editor-variables.png)
+* ![](/.gitbook/assets/interface/logic-editor/arrange.PNG) 
+**Arrange horizontally** and **Arrange vertically** icons will arrange a **Node Graph** horizontally and vertically respectively in the **Logic Editor**. See Usage example below: ![](/.gitbook/assets/interface/logic-editor/arrange-horizontally.gif) ![](/.gitbook/assets/interface/logic-editor/arrange-vertically.gif) 
 
-The **Variables** tab is where you can create variables to be used in your logic. You do this by defining the variable name, selecting the **Data Type** from the drop-down menu and then clicking "Add".
+## 3 - Top Panel
+![](/.gitbook/assets/interface/logic-editor/topPanel.PNG)
 
-You can then drag and drop these variables into the Logic Graph, so that they can be used for getting/setting their values within your logic.
-{% endtab %}
-{% endtabs %}
+The *top panel* represents **Logic Graph**s that are implemented for different Incari items such as the current opened **Project**, the selected **Scene** in the **Project Outliner** and **Functions** added to the **Logic Editor**. Each **Logic Graph** can be accessed by using the tabs on the Top Panel. 
```

Comment:
Inconsistent formatting of top panel. I think the first instance is fine because it's emphasized, however the last instance should be all lower case.